### ISSUE TITLE
TypeScript types

### ISSUE THREAD

**Issue by igeligel** [spencermountain/out-of-character#1]
Hi! 👋 
      
Firstly, thanks for your work on this project! 🙂

Today I used [patch-package](https://github.com/ds300/patch-package) to patch `out-of-character@1.2.0` for the project I'm working on.

The TypeScript definition file was wrong making it unusable to use that package

Here is the diff that solved my problem:

```diff
diff --git a/node_modules/out-of-character/types/index.d.ts b/node_modules/out-of-character/types/index.d.ts
index c9f9573..584da0b 100644
--- a/node_modules/out-of-character/types/index.d.ts
+++ b/node_modules/out-of-character/types/index.d.ts
@@ -1,5 +1,5 @@
 /** find there invisible characters in this text? */
-export function detect(input: string): []object
+export function detect(input: string): object[]
 
 /** remove invisible characters from the text */
 export function replace(input: string): string
```

<em>This issue body was [partially generated by patch-package](https://github.com/ds300/patch-package/issues/296).</em>

**Comment by spencermountain** [comment]
nice find! thank you.
fixed in  1.2.1

**Comment by igeligel** [comment]
Wow thanks a lot!
